app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
```

Comment:
I agree that guessing how to generalise things can sometimes not be ideal but where possible, building interface elements as components is the preferred approach. I'm very much of the school of thought that you 'think in components' (There's plenty of posts with this as their title) and there are just as many articles out there that discuss this approach as there are about premature optimisation.

All I'm asking is, when looking at a new thing look for an existing component/pattern that we hve created and if there isn't one then try to think of it in a general way, not specific to one screen or one feature. With the fields that looks good, with the 50/50 thing I'd come up with a name split 'split-panel' or something.

I agree that Gary needs to be involved, and want him to start looking at the PR's so he's aware of the interface work we are doing so he can reuse the patterns we come up with.

In the projects I've worked on, using the component approach meant that after the initial styles were agreed we rarely needed a designer to create screens or designs, and the interfaces defined themselves based on the needs and existing patterns.